integration-cli/docker_cli_config_test.go

```diff
@@ -141,10 +128,9 @@ func (s *DockerSuite) TestConfigDir(c *check.C) {
 	headers = map[string][]string{}
 	cmd = exec.Command(dockerBinary, "--config", "MissingDir", "-H="+server.URL[7:], "ps")
 	cmd.Env = append(os.Environ(), "DOCKER_CONFIG="+cDir)
-	out, _, _ = runCommandWithOutput(cmd)
+	out, _, err = runCommandWithOutput(cmd)
 
-	if headers["Myheader"] != nil {
-		c.Fatalf("ps6 - Headers are there but shouldn't be: %q\nout:%v", headers, out)
-	}
+	c.Assert(err, checker.NotNil)
```

Comment:
Thanks @cpuguy83 ,and how about lines 101,111,and 121? need a `check.Commentf` or not?